src/java.desktop/share/classes/java/awt/event/KeyEvent.java

```diff
@@ -650,53 +649,142 @@
     public static final int VK_KP_RIGHT       = 0xE3;
 
     /* For European keyboards */
-    /** @since 1.2 */
+    /**
+     * Constant for the DEAD_GRAVE function key.
```

Comment:
I do not know, are all the keys here actually "function keys"?

Reply:
Remove function keyword as it seems dead keys are combination of keys.